tests/pass10/t01_loop.expected

```diff
@@ -0,0 +1,12 @@
+;$sexp
+(TypeDefs
+  (ProcTy (Void)))
+(GlobalDefs)
+(ProcDefs
+  (ProcDef (Type 0)
+    (Continuations
+      (Continuation (Params) (Locals)
+        (Continue 1 (List)))
+      (Continuation (Params) (Locals)
+        (Drop (IntVal 100))
```

Comment:
Sorry for what is undoubtedly a silly question, but I checked out what `opcDrop` does and I can't figure out why we want a `<value>` operand for the op code, it seems unused. Is the idea that in the future it'll be possible to specify the number of items, or the address (?), we want to drop?

Reply:
`opcDrop` pops and discard the last operand from the operand stack, it's purpose is to remove operand from the stack without having to assign them to a dummy local. The `Drop` operation has the same role as `discard`, that is, it evaluates the operand but discards  the result.

`(Drop (IntVal 100))` just means "discard the integer value 100". I've used it as form of "marker", since it's a simple, short, and side-effect-free statement. The idea is to us them to make sure that statements are placed correctly in the continuations after splitting up the statement list(s).